src/main/java/org/reaktivity/nukleus/http2/internal/types/stream/Http2HeadersFW.java

```diff
@@ -80,12 +81,11 @@ public byte flags()
         return buffer().getByte(offset() + FLAGS_OFFSET);
     }
 
+    // streamId != 0, caller to validate
     public int streamId()
     {
         // Most significant bit is reserved and is ignored when receiving
```

Comment:
This comment is unnecessary as it's all defined explicitly in the RFC and/or spec scripts, plus it's already explicit in the code.